pkgs/games/pegasus-frontend/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, cmake
+, libsForQt5
+, wrapQtAppsHook
+, SDL2
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pegasus-frontend";
+  version = "0-unstable-2023-05-22";
+
+  src = fetchFromGitHub {
+    owner = "mmatyas";
+    repo = "pegasus-frontend";
+    rev = "6421d7a75d29a82ea06008e4a08ec14e074430d9";
+    sha256 = "sha256-mwJm+3zMP4alcis7OFQUcH3eXlRTZhoZYtxKrvCQGc8=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libsForQt5.qt5.qtbase
+    libsForQt5.qt5.qtmultimedia
+    libsForQt5.qt5.qtsvg
+    libsForQt5.qt5.qttools
+    libsForQt5.qt5.qtgraphicaleffects
+    libsForQt5.qt5.qtx11extras
```

```suggestion
    qtbase
    qtmultimedia
    qtsvg
    qtgraphicaleffects
    qtx11extras
```
isn't qttools native

---

done